cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js

```diff
@@ -48,9 +48,16 @@ describe('Moderation Tools for Posts', () => {
         cy.findByRole('button', { name: 'Moderation' }).click();
 
         cy.getIframeBody('[title="Moderation panel actions"]').within(() => {
-          cy.findByRole('button', { name: 'Open adjust tags section' }).click({
-            force: true,
-          });
+          cy.findByRole('button', { name: 'Open adjust tags section' })
+            .as('adjustTagsButton')
+            .click({
+              force: true,
+            });
+          cy.get('@adjustTagsButton').should(
```

Comment:
Checking that `aria-expanded` is set to `"true"` means the panel should be open making the tag buttons discoverable to the test.